tests/test_salesforce_custom_fields_test.py

```diff
@@ -0,0 +1,43 @@
+"""
+Test that with no fields selected for a stream automatic fields and custom fields  are still replicated
+"""
+
+from tap_tester.base_suite_tests.all_fields_test import AllFieldsTest
+from sfbase import SFBaseTest
+
+
+class SFCustomFieldsTest(AllFieldsTest, SFBaseTest):
+    """Test that with no fields selected for a stream automatic fields and custom fields are still replicated
+    TODO
+    This test coveres only BULK api and just couple of streams. We have separate cards to cover these cases
+    TDL-23653: [tap-salesforce]: QA - Add all the streams for the all_fields test
+    TDL-23654: [tap-salesforce]: QA - Add all-fields testcase for REST API streams
+    """
+
+    salesforce_api = 'BULK'
+
+    @staticmethod
+    def name():
+        return "tt_sf_custom_fields_test"
+
+    def streams_to_test(self):
+        streams = {'Account', 'Contact'}
+        return streams
+
+    def test_custom_fields( self ):
+        expected_streams = self.selected_fields
+        expected_fields = set()
+        for stream in self.streams_to_test():
+            for value in expected_streams.get(stream, set()):
+                if value.endswith("__c"):
+                    expected_fields.add(value)
```

Comment:
Since we are building a set of fields should we implement a counter to check for any duplicates and assert the count == length of the set?

Also I'd like to use more descriptive names.  Maybe something like:
selected_custom_fields
replicated_custom_fields
or 
expected_custom_fields
actual_custom_fields

Reply:
Renamed the fields with more descriptive names